docs/rules/arazzo/sourceDescriptions-empty.md

```diff
@@ -0,0 +1,119 @@
+# sourceDescriptions-empty
+
+The Source Description object must be defined when `operationId` or `operationPath` is used inside Step to referense description operation.
+`sourceDescriptions` can be undefined in the Arazzo description in case there are no OpenAPI description exist and `x-operation` extension is used inside Step.
+
+| Arazzo | Compatibility |
+| ------ | ------------- |
+| 1.0.0  | ✅            |
+
+## API design principles
```

Comment:
This heading isn't good.

Reply:
any suggestions ?
I took it from some existing examples here https://github.com/Redocly/redocly-cli/pull/1697
But I saw there are no final decision on the format.

Reply:
cc: @lornajane

Reply:
If no suggestion - I can leave it as it is - so we can update heading later.